urllib3/_backends/twisted_backend.py

```diff
@@ -15,8 +15,8 @@
 
 
 class TwistedBackend:
-    def __init__(self, reactor):
-        self._reactor = reactor
+    def __init__(self, reactor=None):
```

Comment:
Ideally all of our backends can be initialized without any arguments, where possible.